daemon/src/projection.rs

```diff
@@ -500,6 +500,8 @@ pub struct Cfd {
 
     #[serde(with = "::time::serde::timestamp")]
     pub expiry_timestamp: OffsetDateTime,
+
+    pub counterparty_identity: Identity,
```

Comment:
don't you need to also add the Identity in the both frontends to be displayed/parsed correctly? will it not break the UI as it is (have you tested both maker and taker?)

`interface Cfd` would probably require changes it 2 places, as we copied the file with creation of the new frontend :(

Reply:
It will just be ignored in the UI.

Reply:
good to know! but maybe it would be worthwhile to keep them in sync? (we don't need to display them rn, we do it already for a few fields)